PBLV5/Core/Inc/globalDefines.h

```diff
@@ -17,6 +17,7 @@
 #define ID_GPS 0x08
 #define ID_MINI_LIDAR 0x09
 #define ID_RULER 0x12
+#define ID_SIDE_RULER 0x13
```

Comment:
Keep in mind changing its id before merging (conflict with currentMeasurement branch)